units/emergency.service.in

```diff
@@ -19,7 +19,7 @@ Before=rescue.service
 [Service]
 Environment=HOME=/root
 WorkingDirectory=-/root
-ExecStartPre=-/bin/plymouth --wait quit
+ExecStartPre=-@rootbindir@/plymouth --wait quit
```

Comment:
I think it is a valid question: why muck with this, when ExecStartPre=plymouth works just as well? This is obviously non-trivial to get right, and thing would be just simpler and more robust if we don't insist on a fixed path.

Reply:
still not a fan of searching in $PATH... we know where the thing is, it can't really be anywhere else than `@rootbindir@`, except if the distros is borked...

Reply:
(but anyway, i don't care much, would be happy to merge a patch that drops the prefix too)

Reply:
OK, so let's merge this then.